index.html

```diff
@@ -63,74 +63,7 @@
     <!-- About us: end -->
 
     <!-- Our services: start -->
-    <section class="container">
-      <div class="row">
-        <div class="col-12">
-          <span>Our Solutions</span>
-          <h2>Our Services</h2>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incididunt ut labore et dolore magna aliqua.
-          </p>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">IT Profesionals</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">Software Engineers</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">Web Development</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">SEO & Content</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">Digital Marketing</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-        <div class="col-12 col-sm-6 col-md-6 col-lg-4">
-          <img src="#" alt="photo" />
-          <div>
-            <h3>
-              <a href="#">Data Analysts</a>
-            </h3>
-            <a href="#">Read More</a>
-          </div>
-        </div>
-      </div>
-    </section>
```

Comment:
bijau, kad gali issitrinti approvinus sita requesta, gal is develop viska susikelt reikia ir nauja pull request padaryti?

Reply:
old stuff